src/unistd.rs

```diff
@@ -2459,6 +2459,45 @@ impl From<*mut libc::passwd> for User {
 }
 
 impl User {
+    fn from_anything(f: impl Fn(*mut libc::passwd,
```

Comment:
This signature is so big that it would probably be written better with a where clause:
```rust
    fn from_anything<F>(f: F) -> Result<Option<Self>>
    where
        F: Fn(
            *mut libc::passwd,
            *mut libc::c_char,
            libc::size_t,
            *mut *mut libc::passwd,
        ) -> libc::c_int
```